7.4/fpm/ubuntu/Dockerfile

```diff
@@ -31,6 +40,7 @@ RUN apt-get update \
         php7.4-ctype \
         php7.4-zip \
         php7.4-bcmath \
+        php7.4-dev \
```

Comment:
I'm a bit worried about this dependency. It will probably add some additional MBs to the image only to be used when one wants to build it w/ xdebug. I'd move this to the `install_xdebug.sh`

Reply:
Done.